README.md

```diff
@@ -184,19 +191,19 @@ rake local_authority:create -- --subdomain 'lambeth' \
   --feedback_email 'mail@lambeth.gov.uk' \
   --council_name 'Lambeth Council' \
   --short_name 'Lambeth' \
-  --applicants_url 'https://planningapplications.lambeth.gov.uk' \
+  --applicants_url 'https://lambeth.planningapplications.gov.uk' \
```

Comment:
The applicant's production url is correct, as is directed to the subdomain `planningapplications` of lambeth main website, thus is `https://planningapplications.lambeth.gov.uk`

Reply:
Ah, good catch thanks. Reverted.